ktesting/init/init.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package init registers the command line flags for k8s.io/klogr/testing in
+// the flag.CommandLine. This is done during initialization, so merely
+// importing it is enough.
+//
+// Experimental
+//
+// Notice: This package is EXPERIMENTAL and may be changed or removed in a
+// later release.
+package init
+
+import (
+	"flag"
+
+	"k8s.io/klog/v2/ktesting"
+)
+
+func init() {
+	ktesting.DefaultConfig.AddFlags(flag.CommandLine)
```

Comment:
I'm not sure about registering testing flags in init. Have we talked with SIG testing about what is the best way to integrate logging into test framework?

Reply:
This is a Golang problem, not a SIG Testing problem. I asked Tim and he suggested to ask on golang-nuts. I [did that](https://groups.google.com/g/golang-nuts/c/0hc2qIyFSuw/m/NR4NxYxnAQAJ) but got no responses.

Reply:
I've also asked on SIG Testing, just in case that someone there has an opinion: https://kubernetes.slack.com/archives/C09QZ4DQB/p1646918432994899